frontend/src/components/GoodsItem.tsx

```diff
@@ -9,6 +9,15 @@ export interface Props {
   image_url: string;
 }
 
+export interface GoodsItemProps {
+  description: string;
+  id: number;
+  image_url: string;
+  name: string;
+  price: number;
+  sales: number;
+}
```

Comment:
i know you mentioned the attributes of GET `/seller/product` is different from this
actually, in each goodsItem, I only need (a) id (b) name (c) image_url, basically the left attributes I won’t use
the reason I put it in GoodsItem.tsx is because this interface or type belongs to it, putting in home may seem like classification error
i’ll maybe change name of interface for later use